pages/login/profile/[id].js

```diff
@@ -21,6 +25,11 @@ export default function DetailProfile() {
   if (isLoading) return <h1>Loading</h1>;
   if (!data) return <h1>Den Kaffee gibts wohl nicht ...</h1>;
 
+  function showClickedRecipe(id) {
+    setShowModal(true);
+    setPickedRecipe(...data.relatedMethods.filter(({ _id }) => _id === id));
```

Comment:
I guess you can only find one element here, then you should use `.find` instead of `.filter` and remove the spread operator